src/Stratis.Bitcoin/P2P/PeerAddressManagerBehaviour.cs

```diff
@@ -68,15 +77,32 @@ protected override void AttachCore()
 
         private async Task OnMessageReceivedAsync(INetworkPeer peer, IncomingMessage message)
         {
+            this.logger.LogTrace("({0}:[{1}], {2}:[{3}])", nameof(peer), peer, nameof(message), message);
```

Comment:
this.logger.LogTrace("({0}:'{1}',{2}:'{3}')", nameof(peer), peer.RemoteSocketEndpoint, nameof(message), message.Message.Command);